core/jazz_ui/src/app/pages/environment-logs/env-logs-section.component.ts

```diff
@@ -189,18 +188,23 @@ export class EnvLogsSectionComponent implements OnInit {
 		(<AdvancedFiltersComponent>componentRef.instance).onFilterSelect.subscribe(event => {
 			comp.onFilterSelect(event);
 		});
-		this.instance_yes.onAssetSelect.subscribe(event => {
-		
+		this.instance_yes.onAssetSelect.subscribe(event => {		
 			comp.onAssetSelect(event);
 		});
+		this.instance_yes.onResourceSelect.subscribe(event => {
```

Comment:
Don't forget to unsubscribe on component teardown. See [this](https://blog.codecentric.de/en/2018/01/different-ways-unsubscribing-rxjs-observables-angular/).